### ISSUE TITLE
clean code

### ISSUE THREAD

**Issue by ThatNerdyPikachu** [PretendoNetwork/maryo#2]
tbh, it's not perfect, and some modifications could be made to make it easier to read or something idk

**Comment by superwhiskers** [comment]
true, i was going to do this during the rewrite but other things have been taking up my time

**Comment by ThatNerdyPikachu** [comment]
*cough* **raiding servers** *cough*
